client/components/status/status.js

```diff
@@ -172,12 +173,12 @@ export default class Status extends React.Component {
   };
 
   async finalOperations() {
-    const {userData, settings} = this.props;
+    const {userData, orgSlug, settings} = this.props;
     const {setLoading} = this.context;
     // if the user needs bank card verification,
     // redirect to payment page and stop here
     if (needsVerify("bank_card", userData, settings)) {
-      window.location.assign(userData.payment_url);
+      redirectToPayment(orgSlug);
```

Comment:
This should redirect to the payment process component, which should know whether to load the payment_url in an iframe or just redirect to it, depending on configuration.